### DISCUSSION TITLE
Customize log in/out and authorize pages

### DISCUSSION THREAD

**Discussion by dor1202** [OrchardCMS/OrchardCore#11771] [Q&A]
Hello.
I'm fairly new to orchardCore, I'm using the open id for authorization but I want to customize the above pages and create a new layout and styling.
I can customize the login when using:
`@model OrchardCore.Users.ViewModels.LoginViewModel`

But I couldn't customize the log out and authorize pages when using:
`@model OrchardCore.OpenId.ViewModels.LogoutViewModel`
`@model OrchardCore.OpenId.ViewModels.AuthorizeViewModel`

The file tree is:
Views
--OrchardCore.OpenId
----Logout.cshtml
----Authorize.cshtml
--OrchardCore.Users
----Login.cshtml

Am I missing something?

**Comment by vitalybrandes** [comment]
https://github.com/OrchardCMS/OrchardCore/issues/11644
